domain-check-2.sh

```diff
@@ -409,6 +412,10 @@ check_domain_status()
     then
        ${WHOIS} -h whois.isnic.is "${1}" | env LC_CTYPE=C LC_ALL=C ${TR} -d "\r" > ${WHOIS_TMP}
     fi
+    if [ "${TLDTYPE}" == "ca" ]; # added by @hawkeye116477 20190511
+    then
+       ${WHOIS} -h whois.ca.fury.ca "${1}" | env LC_CTYPE=C LC_ALL=C ${TR} -d "\r" > ${WHOIS_TMP}
```

Comment:
No need to change whois server.
My whois query works correctly:
```
...
+ /usr/bin/whois -h whois.iana.org ca
+ /usr/bin/grep whois:
+ /usr/bin/awk '{print $2}'
+ WHS=whois.cira.ca
+ [ ca '==' jp ]
+ /usr/bin/whois -h whois.cira.ca google.ca
+ env 'LC_CTYPE=C' 'LC_ALL=C' /usr/bin/tr -d '\r'
...
```

Reply:
Ok, changed :smile: